packages/calcite-components/src/components/avatar/readme.md

```diff
@@ -1,7 +1,8 @@
 # calcite-avatar
 
-The avatar component provides a unique (but consistent) visual for a user. It's built
-to work with users returned from the ArcGIS REST API.
+The `calcite-avatar` component serves as a visual representation of a user's account. It can be displayed using either an image or initials.
+
+A randomly generated AA accessible background will fill behind the initials. That generated fill will stay assigned to the user until their account is deleted or an image is used.
```

Comment:
This is an implementation of the app (persisting for user) - I'd remove all but the first sentence.